frontend/src/containers/PublicPage.js

```diff
@@ -18,7 +18,7 @@ export class PublicPage extends Component {
       return <ProfilePage profile={ profile } />
     } else if (this.props.showOldCollectivePage) {
       return <PublicGroup />
-    } else if (this.props.group.slug ) {
+    } else if (this.props.slug ) {
```

Comment:
Never mind, I ended up using a separate static website for BrusselsTogether.org.